packages/core/bootstrap/src/lib/external-adapter/requester.ts

```diff
@@ -46,17 +46,18 @@ export class Requester {
       }
 
       let response: AxiosResponse<T>
+      const endpoint = `${config.baseURL}/${config.url}`
```

Comment:
The endpoint should come from the request parameter `endpoint`.
`.url` won't be set in most places.

Reply:
Though this might still be useful information. I would keep this, but rename it `url`.

Reply:
We don't have access to endpoint parameter.

Reply:
resolved in #1058